js/script.js

```diff
@@ -968,6 +971,9 @@ var validCombos = {
     "android,C++,ARM64,NNAPI":
         "Follow build instructions from <a href='https://aka.ms/build-ort-nnapi' target='_blank'>here</a>",
     
+    "android,C#,ARM64,NNAPI":
+        "Follow build instructions from <a href='https://aka.ms/build-ort-nnapi' target='_blank'>here</a>",
+    
```

Comment:
For C# they really need to use the nuget packages as there's no good way to build manually. There's also no reason to.

Reply:
Updated all C# + Android packages. The instructions all point to the Xamarin sample...is that correct?

Reply:
I would just point a user at the [C# nuget package](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime)

Reply:
Updated again. If it all looks good now, can you sign off?